[2021-12-13 11:41:53,932] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.check_imageFile manual__2021-12-13T11:41:51.288728+00:00 [queued]>
[2021-12-13 11:41:53,962] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.check_imageFile manual__2021-12-13T11:41:51.288728+00:00 [queued]>
[2021-12-13 11:41:53,963] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-13 11:41:53,963] {taskinstance.py:1242} INFO - Starting attempt 1 of 3
[2021-12-13 11:41:53,964] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-13 11:41:54,001] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): check_imageFile> on 2021-12-13 11:41:51.288728+00:00
[2021-12-13 11:41:54,015] {standard_task_runner.py:52} INFO - Started process 1000 to run task
[2021-12-13 11:41:54,020] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cpp_workflow', 'check_imageFile', 'manual__2021-12-13T11:41:51.288728+00:00', '--job-id', '895', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpnt0apyj9', '--error-file', '/tmp/tmpf2t264ud']
[2021-12-13 11:41:54,031] {standard_task_runner.py:77} INFO - Job 895: Subtask check_imageFile
[2021-12-13 11:41:54,156] {logging_mixin.py:109} INFO - Running <TaskInstance: cpp_workflow.check_imageFile manual__2021-12-13T11:41:51.288728+00:00 [running]> on host 218df5ddc363
[2021-12-13 11:41:54,335] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cpp_workflow
AIRFLOW_CTX_TASK_ID=check_imageFile
AIRFLOW_CTX_EXECUTION_DATE=2021-12-13T11:41:51.288728+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-13T11:41:51.288728+00:00
[2021-12-13 11:41:54,338] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-12-13 11:41:54,339] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sha256sum /opt/***/official_data/Image.csv']
[2021-12-13 11:41:54,366] {subprocess.py:85} INFO - Output:
[2021-12-13 11:41:54,387] {subprocess.py:89} INFO - e891870a24e5a22c9520c5d417af923366083a3617288cafa3cc18e3fed77b11  /opt/***/official_data/Image.csv
[2021-12-13 11:41:54,387] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-13 11:41:54,469] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=cpp_workflow, task_id=check_imageFile, execution_date=20211213T114151, start_date=20211213T114153, end_date=20211213T114154
[2021-12-13 11:41:54,533] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-13 11:41:54,665] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
